<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddScene</class>
 <widget class="QDialog" name="AddScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Постановка</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labeName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>61</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Название</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="NameLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>251</width>
             <height>31</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="labelDateTime">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>81</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Дата и время</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QDateEdit" name="dateEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>101</width>
               <height>31</height>
              </size>
             </property>
             <property name="displayFormat">
              <string>dd.MM.yyyy</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
             <property name="date">
              <date>
               <year>2016</year>
               <month>1</month>
               <day>1</day>
              </date>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="timeEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>101</width>
               <height>31</height>
              </size>
             </property>
             <property name="currentSection">
              <enum>QDateTimeEdit::HourSection</enum>
             </property>
             <property name="displayFormat">
              <string>HH:mm</string>
             </property>
             <property name="calendarPopup">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBoxPrice">
         <property name="minimumSize">
          <size>
           <width>361</width>
           <height>91</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>Цена, грн</string>
         </property>
         <widget class="QDoubleSpinBox" name="ParterPrice">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>101</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="BenuarPrice">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>50</y>
            <width>101</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="BeletazPrice">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>50</y>
            <width>101</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Партер</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>30</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Бенуар</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>30</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Бельэтаж</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonAdd">
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>51</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>51</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
